### PR TITLE
(docs): link to discourse in README

### PR DESCRIPTION
**Pull request by zoechi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ As of February 2020, it is in a very early stage of development.
 Important links:
 
 - **[Documentation][docs]**
-- **[Org-roam Slack][slack]**
+- **[Org-roam Discourse][discourse]**
```
Comment (zaeph) [inline README.md:30]: I don't think we should be removing the Slack.  It's still useful for live support.

File: README.md [hunk 2]
```diff
@@ -64,6 +64,12 @@ Here's a sample configuration with using `use-package`:
               (("C-c n i" . org-roam-insert))))
 ```
 
+`org-roam-graph` by default expects to find the `dot` executable 
+from the `graphviz` package in the `$PATH`.
+Ensure `graphviz` is installed if you want to use this feature
+or customize your configuration for `org-roam-graph` to use a 
+different tool.
```
Comment (jethrokuan) [inline README.md:71]: This section is not entirely correct. It doesn't have to be in env variable `$PATH`, but it needs to be in Emacs-lisp list `exec-path`. Ensuring graphviz is installed doesn't make the path to the executable available to Emacs either, that requires a bit more careful customization, but is typically resolved with package `exec-path-with-shell`.

File: README.md (hunk 1 above)
Comment (zoechi) [reply README.md:30]: Ok, I thought new users shouldn't be directed there. I'll fix.

File: README.md (hunk 2 above)
Comment (zoechi) [reply README.md]: Thanks for the suggestion. I changed it a bit.

File: README.md [hunk 3]
```diff
@@ -28,6 +28,7 @@ Important links:
 
 - **[Documentation][docs]**
 - **[Org-roam Discourse][discourse]**
```
Comment (jethrokuan) [inline README.md:30]: Could you also change this to just "Discourse"? seems a bit more consistent.

**Pull request merged**